portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,17 +34,26 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  
   private static final Gson GSON = new Gson();
-  private ArrayList<String> comments;
-
-  @Override
-  public void init() {
-    comments = new ArrayList<>();
-  }
+  private static final String COMMENT_KEY = "comment";
+  private static final String TIMESTAMP_KEY = "timestamp";
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
make "Comment" and "timestamp" static final

Reply:
done.